src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,118 +1,132 @@
-<h1>Add Courses</h1>
+<h1 i18n>Add Courses</h1>
 <!-- nvalidate - disable browser's default validators to use our validators -->
 <form [formGroup]="courseForm" (ngSubmit)="onSubmit()" novalidate>
   <div class="row">
     <div class="form-group col-4">
-      <label>Course Title</label>
-      <input type="text" class="form-control" formControlName="courseTitle" required>
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Course Title" formControlName="courseTitle" required>
+      </mat-form-field>
       <planet-form-error-messages [control]="courseForm.controls.courseTitle"></planet-form-error-messages>
     </div>
     <div class="form-group col-4">
-      <label>Language of Instruction</label>
-      <input type="text" class="form-control" formControlName="languageOfInstruction">
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Language of Instruction" formControlName="languageOfInstruction" required>
+      </mat-form-field>
     </div>
     <!-- Firefox seems to accept non-numeric characters but it does not actually change the value of the DOM -->
     <div class="form-group col-4">
-      <label>Member Limit</label>
-      <input type="number" min="1" class="form-control" formControlName="memberLimit">
+      <mat-form-field>
+        <input matInput i18n-placeholder placeholder="Member Limit" type="number" class="example-right-align" formControlName="memberLimit">
+      </mat-form-field>
       <planet-form-error-messages [control]="courseForm.controls.memberLimit"></planet-form-error-messages>
     </div>
   </div>
   <div class="row">
     <div class="form-group col-4">
-      <label>Course Leader</label>
+      <mat-form-field>
+        <mat-select i18n-placeholder placeholder="Course Leader" formControlName="courseLeader">
+          <mat-option *ngFor="let memb of members" value={{memb?.id}}> {{memb?.doc.firstName}} {{memb?.doc.middleName}} {{memb?.doc.lastName}}</mat-option>
+        </mat-select>
+      </mat-form-field>
       <!-- need to create a select with options from number of members available -->
```

Comment:
why is all this comment still here when there is no option field used here

Reply:
the commet is madeto remind  so that in future while working in this course section members will be fetched from member database

Reply:
you are already giving option of members here

Reply:
this isssue is only to conver course form into material, we will create new isssue for this one.